docs/source/hardware/remote_controllers.rst

```diff
@@ -1,3 +1,104 @@
 ==================
 Remote Controllers
 ==================
+
+.. meta::
+    :description lang=en: Hardware page for the quadrotor's remote control transmitter.
+
+The quadrotors require a standard remote control transmitter to 
+
+* arm the flight controller, 
+* select flight control modes, 
+* fly manually, and
+* perform emergency landings. 
+
+We use the Spektrum DX6e 6-channel transmitter. This page documents the configuration of this transmitter and its use with the quadrotors. 
+
+Configurations
+------------------
+
+There are three transmitter features to be familiar with: 1) the control sticks, 2) the flight mode toggle, and 3) the arm/disarm toggle and the throttle cut toggle. 
+
+**Control Sticks**
+The control stick control the throttle, roll, pitch, and yaw of the quadrotor. The *zero configuration* is illustrated in the image below where all sticks are at the center of their range. 
+
+**Flight Mode Toggle**
+The flight mode toggle switches between stability mode (0), position hold mode (1), and offboard mode (2). 
+
+**Arm/Disarm Toggle**
+The arm/disarm toggle arms (1) or disarms (0) the quadrotor. The optional throttle cut switch cuts the throttle if moved from the (0) position. This is a last resort option to stop the motors from spinning. *DO NOT* cut throttle in flight as the quadrotor will drop to the ground. 
```

Comment:
Would the DO NOT part be better as a warning or bolded? Warning could be done with 
```
.. warning::
   Warning message
```
https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions